package.json

```diff
@@ -33,7 +33,7 @@
     "release": "node scripts/release --darwin",
     "prerelease": "npm run build",
     "prebuild": "npm run clean",
-    "postinstall": "npm-run-all -p deps:*",
+    "postinstall": "npm-run-all -s deps:*",
```

Comment:
Running these parallel makes it hard to know why an npm install is taking so long.